index.html

```diff
@@ -9,24 +9,44 @@
   <script defer src="index.js"></script>
 </head>
 <body>
-  <div class="container">
-    <header>
-      <h1 class="heading-title">All awesome books</h1>
-    </header>
-    <section id="books-collection"></section>
-    <div class="divide"></div>
-    <section>
-      <h4 class="form-heading">Add a new book</h4>
-      <form id="form" action="">
-        <div>
-          <input type="text" id="title-input" class="input-area" placeholder="Title" required autofocus>
-        </div>
-        <div>
-          <input type="text" id="author-input" class="input-area" placeholder="Author">
-        </div>
-        <button type="submit" class="form-button">Add</button>
-      </form>
-    </section>
+  <header class="page-header">
+    <span class="logo">Awesome Books</span>
+    <nav class="navigation">
+      <span class="nav-link listBtn" id="list-books">List |</span>
+      <span class="nav-link add-new-btn" id="add-new-book">Add new |</span>
+      <span class="nav-link contact" id="contact">Contact</span>
+    </nav>
+  </header>
+  <div>
+    <p id="set-date"></p>
   </div>
+  <main>
+    <div class="container">        
+      <section id="books-section">
+        <h2 class="heading-title">All awesome books</h2>
+        <div id="books-collection" class="hide"></div>
+      </section>
+        <form id="form" action="" class="form-container">
+          <h2 class="heading-title">Add a new book</h2>
+            <input type="text" id="title-input" class="input-area" placeholder="Title" required autofocus>
+            <input type="text" id="author-input" class="input-area" placeholder="Author">
+          <button type="submit" class="form-button">Add</button>
```

Comment:
- [ ] [optional] when I type something on the inputs I don't see what I'm typing which will confuse the user please make sure to fix this in order to improve the UX